include/common_components.php

```diff
@@ -1,24 +1,384 @@
                 <title>$title</title>  
                 <link href='style.css' rel='stylesheet'>                                                          
-            </head>    
-            <body>
+            </head>            
+            <body>";
+}
+
+// --------------------------------------------------------------------------
+
+function echoVerticalBar(){
+    echo"
+        <section class='wrapper'>
+            <section>  
+                <div class='flex-container-verticalBar'>  
+                    <div class='flex-item-verticalBarPicture'>
+                        <img class ='circlePicture' src= '/images/Profile_Yuni.jpg' alt='Profile_Yuni'>
+                        <h1 >Yuniesky Quesada</h1>
+                    </div>            
+                </div>  
+        </section>      
     ";
+  }
+
+// --------------------------------------------------------------------------
+
+function echoHorizontalBar($heading, $subheading){
+ echo " 
+    <section>     
+            <nav>
+                <header>$heading
+                    <p> $subheading</p>
+                </header>
+                <div class='flex-item-horizontalMenu'>
+                    <ul>                         
+                        <a href='#'> <img class = 'notificationIcon' src= '/images/notification.png' alt='notification'></a> 
+                        <li><a href='#'>Students</a></li>
+                        <li><a href='#'>Calendar</a></li> 
+                        <li><a href='#'>Home</a></li>                        
+                    </ul> 
+                </div>                
+            </nav>      
+";
 }
 
+// --------------------------------------------------------------------------
+
 function echoFooter(){
-    echo"  <footer></footer>                            
+    echo"  
+                    </section>
+                </section>  
+                <footer></footer>                            
            </body>    
         </html>
        ";
+}
+
+// list_students.php file
+// --------------------------------------------------------------------------
+
+function echoSearchBar($heading){
+    echo"
+    <div class='flex-container-searchBarButtons'>
+        <div>
+            <h2>$heading</h2>
+            <div class='flex-item-SearchBar'>
+                <form action='' class='search-bar'>
+                <input type='text' placeholder='Search' name='searchBar'>
+                <button class='searchB' type='submit'><img class= 'searchButton'src='/images/search.jpg' alt='search'></button>
+                </form>
+            </div> 
+        </div>     
+";    
+}
+
+// --------------------------------------------------------------------------
+
+function addButtons($heading){
+    echo"
+        <div class='flex-item-buttons'>              
+            <button onclick= document.getElementById('id01').style.display='block' style='width:fit-content;'>$heading</button>
+        </div>
+    </div>    
+";    
+}
```

Comment:
About the schema: I didn't realized the DB schema file was being submitted sorry. I copied it in the folder but yep, I see it submits everything in the folder lol. This is how I learned at school and I LOVE those haha. So I was using it as reference the first couple of days when I was getting familiar with my very tiny DB.
About the images: Yes, I need to handle that and also how to store them etc.
About the function: Yes, it echoes the Add Student button that when clicked displays a pop up form (it does not go to a different page) to add a new student. I know what you mean with dynamically but I'm not sure I'm understanding what you mean in this context. 
I also changed the name of the function to echoAddStudentButton so is more descriptive. Tyler was telling me also that this might not need to be a function but I sort of like to have them because it helps me keep things organize since every function does one thing. By just reading the functions' names (if named correctly), you can get an idea of what's happening with the code. I have to improve the names tho